flink-cdc-runtime/src/main/java/com/ververica/cdc/runtime/operators/schema/event/FlushEvent.java

```diff
@@ -28,8 +28,7 @@
  */
 public class FlushEvent implements Event {
```

Comment:
Not a `OperatorEvent`?  We should distinguish the flink operator event and  cdc event, the later should put on `common` module and add `@PublicEvolving` if it expose to connector developers.

Reply:
The `FlushEvent` is not an `OperatorEvent`. I will move it to the common module.